packages/connect-query/src/use-query.test.ts

```diff
@@ -123,6 +123,42 @@ describe("useQuery", () => {
 
     expect(result.current.data).toBe(6);
   });
+
+  it("can be disabled without explicit disableQuery", () => {
+    const { result } = renderHook(
+      () => {
+        return useQuery(
+          sayMethodDescriptor,
+          {
+            sentence: "hello",
+          },
+          {
+            enabled: false,
+          },
+        );
+      },
+      wrapper({}, mockedElizaTransport),
+    );
+
+    expect(result.current.data).toBeUndefined();
+    expect(result.current.isPending).toBeTruthy();
+    expect(result.current.isFetching).toBeFalsy();
```

Comment:
Just for my own knowledge, how do we tell that the query was disabled from these values?

Reply:
There's currently no explicit indicated of disabled with react-query, but basically if it's `isPending` and `isFetching` is false, then the indication is that it's disabled. If it's enabled, the initial state will be `isPending` is true and `isFetching` = true.